tuf-spec.md

```diff
@@ -1238,66 +1246,66 @@ of the form VERSION_NUMBER.FILENAME.EXT (e.g., 42.targets.json), where
 VERSION_NUMBER is the version number of the targets metadata file listed in the
 snapshot metadata file.
 
-  * **5.4.1**. **Check against snapshot role's targets hash.** The hashes
+  * **5.5.1**. **Check against snapshot role's targets hash.** The hashes
   of the new targets metadata file MUST match the hashes, if any, listed in the
   trusted snapshot metadata.  This is done, in part, to prevent a mix-and-match
   attack by man-in-the-middle attackers.  If the new targets metadata file does
   not match, discard the new target metadata, abort the update cycle, and
   report the failure.
 
-  * **5.4.2**. **Check for an arbitrary software attack.** The new targets
+  * **5.5.2**. **Check for an arbitrary software attack.** The new targets
   metadata file MUST have been signed by a threshold of keys specified in the
   trusted root metadata file.  If the new targets metadata file is not signed
   as required, discard it, abort the update cycle, and report the failure.
 
-  * **5.4.3**. **Check against snapshot role's targets version.** The version
+  * **5.5.3**. **Check against snapshot role's targets version.** The version
   number of the new targets metadata file MUST match the version number listed
   in the trusted snapshot metadata.  If the versions do not match, discard it,
   abort the update cycle, and report the failure.
 
-  * **5.4.4**. **Check for a freeze attack.** The latest known time MUST be
-  lower than the expiration timestamp in the new targets metadata file.  If so,
-  the new targets metadata file becomes the trusted targets metadata file.  If
-  the new targets metadata file is expired, discard it, abort the update cycle,
-  and report the potential freeze attack.
+  * **5.5.4**. **Check for a freeze attack.** The expiration timestamp in the
+  new targets metadata file MUST be higher than the fixed update expiration
+  time.  If so, the new targets metadata file becomes the trusted targets
+  metadata file.  If the new targets metadata file is expired, discard it,
+  abort the update cycle, and report the potential freeze attack.
 
-  * **5.4.5**. **Persist targets metadata.** The client MUST write the file to
+  * **5.5.5**. **Persist targets metadata.** The client MUST write the file to
   non-volatile storage as FILENAME.EXT (e.g. targets.json).
 
-  * **5.4.6**. **Perform a pre-order depth-first search for metadata about the
+  * **5.5.6**. **Perform a pre-order depth-first search for metadata about the
   desired target, beginning with the top-level targets role.**  Note: If
   any metadata requested in steps 5.4.6.1 - 5.4.6.2 cannot be downloaded nor
```

Comment:
```suggestion
  any metadata requested in steps 5.5.6.1 - 5.5.6.2 cannot be downloaded nor
```

Reply:
Well spotted, thank you! I've squashed that change into c2c4308